eng/common/scripts/Package-Properties.ps1

```diff
@@ -130,6 +130,17 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
         {
             $filePath = Resolve-Path (Join-Path $RepoRoot $file)
             $shouldInclude = $filePath -like "$pkgDirectory*"
+            $pathComponents = $file -split "/"
```

Comment:
Perhaps it is just me but I prefer a regex for this matching. Maybe we should share the one on line 216 https://github.com/Azure/azure-sdk-tools/pull/9117/files#diff-4958f9a6af9bad851c95447bb46695e45d0f344a6125a5fee7791dd1d36e3a8cR216

Reply:
Ok. I will say that I'm certain this is probably _safer_ than regex, I'll conform to a standard.

Reply:
I tried the regex and it just returns the path as is. Going to continue with what works.

Reply:
![image](https://github.com/user-attachments/assets/5b546b2b-dbf8-411f-b98b-b8fe02c9f86f)